src/components/PostDetails.tsx

```diff
@@ -1,99 +1,53 @@
 import React, { useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import {
+  fetchComments,
+  deleteComment,
+  addComment,
+} from '../features/comments/commentsSlice';
 import { Loader } from './Loader';
 import { NewCommentForm } from './NewCommentForm';
-
-import * as commentsApi from '../api/comments';
-
 import { Post } from '../types/Post';
-import { Comment, CommentData } from '../types/Comment';
+import { CommentData } from '../types/Comment';
```

Comment:
Ensure that the `CommentData` type is correctly defined in the `types/Comment` file.